spec/lib/index.spec.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable max-nested-callbacks */
+
+import path from "path"
+
+import _ from "lodash";
+
+import Interlock from "../../lib/index.js";
+
+const minimalValidConfig = {
+  emit: ["./index.js"],
+  root: __dirname + "/../..",
+};
+
+describe("lib/index.js", () => {
+  describe("constructor", function () {
+    // TODO: Test for [] and undefined. _.merge ignores those values.
```

Comment:
I was shocked to find the following. Seems like a bug to me but I haven't dug in yet...
```
let myOptions = {myArray: [1, 2, 3]}
_.merge(myOptions, { myArray: undefined })
//=> {myArray: [1, 2, 3]}

_.merge(myOptions, { myArray: [] })
//=> {myArray: [1, 2, 3]}
```